doc/Requirements Specification/Parts/Functional Requirements.lyx

```diff
@@ -838,14 +806,21 @@ gls{PCM}
 
 \end_inset
 
- instance provided to Beagle will be found in the 
-\begin_inset CommandInset ref
-LatexCommand ref
-reference "result PCM"
+.
+ The modified 
+\begin_inset ERT
+status open
+
+\begin_layout Plain Layout
+
+
+\backslash
+gls{PCM}
+\end_layout
 
 \end_inset
 
-.
+ will only be extended.
```

Comment:
Does this sound like extending the the model itself (e.g. adding components) for you?
Do you think "... will only be extended by measurement results and parameters" works better?